source/app/views/game.erb

```diff
@@ -0,0 +1,30 @@
+  <h1 id="header"> Let's start! Players, pick your emoji!
+    <hr></h1>
+    <br>
+
+    <br>
+<div class="success">
+
+
+
+    <form id='MyGame' action="/game" method="post">
```

Comment:
We can dry this up further but looping over players and creating each div for the players we pass to the view.